emrun.py

```diff
@@ -1643,10 +1643,6 @@ def run():
     url = file_to_serve
   else:
     url = os.path.relpath(os.path.abspath(file_to_serve), serve_dir)
```

Comment:
Maybe move these 4 lines down too do avoid have two separate `if file_to_serve_is_url` block.

Reply:
Yea so I tried this but the combination of relative and absolute paths to the file to server didn’t play well